compiler/modules/front.syntax/parseStmt.bal

```diff
@@ -222,14 +225,39 @@ function parseCompoundAssignStmt(Tokenizer tok, LExpr lValue, CompoundAssignOp o
     return stmt; 
 }
 
+function parseDestructuringAssignStmt(Tokenizer tok) returns DestructuringAssignStmt|err:Syntax {
+    BindingPattern pattern = check parseBindingPattern(tok);
+    check tok.expect("=");
+    Expr expr = check parseExpr(tok);
+    DestructuringAssignStmt stmt = { pattern, expr };
+    check tok.expect(";");
+    return stmt;
+}
+
+function parseBindingPattern(Tokenizer tok) returns BindingPattern|err:Syntax {
+    Token? cur = tok.current();
+    check tok.advance();
+    if (cur == "_") {
+        return WildcardBindingPattern;
+    }
+    return parseError(tok, "unsupported binding pattern"); // TODO: check the message
```

Comment:
I think `tok.expect("_");` is sufficient, we are not doing custom error messages.

Reply:
This part is removed